batch/batch/worker/worker.py

```diff
@@ -841,7 +841,12 @@ async def delete_container(self):
                 log.info(f'{self} container is still running, killing crun process')
                 self.process.terminate()
```

Comment:
These lines still make me uneasy, because `self.process.terminate()` should be sending a SIGTERM. I am almost inclined to remove this line or put it as a later resort. So
1. Try to ask crun to stop the container with `crun kill`
2. If the above fails, try to `terminate` the crun process itself (it's unclear to me if we should do some sort of waiting to check whether the process responded to the terminate. I'm not sure what there is left to do at this point if it didn't other than log that we tried really hard and didn't succeed)
3. Finally set `process = None`

What do you think?

Reply:
I think this seems reasonable, but I changed the signal for `crun kill` to SIGKILL as I think responding to a SIGTERM is optional. Maybe we call `crun kill` twice with two different signals.